editor/components/upload/upload.ts

```diff
@@ -17,7 +17,7 @@ export class Upload {
 
     handleFileChange(event: Event) {
         let files = (event.target as HTMLInputElement).files;
-        let file: File = files && files[0];
+        let file = files && files[0];
```

Comment:
I removed the type annotation because it wasn't necessary (typescript is inferring the type correctly) and incorrect when typescript is run with `strictNullChecks` as `files[0]` might be null. The inferred type is `File | undefined`